source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,66 +1,74 @@
+The mixer tool is used to create a customized image based on |CL-ATTR|. This is
+known as a *mix*. Creating a custom mix provides a high degree of control and
+customization, but requires you to act as your own OSV. This guide covers the
+basic workflow of creating your own mix using the mixer tool and setting up a
+basic web server to host your mix for downstream users.
 
 Prerequisites
 *************
 
-To start working with the mixer tools, you need a recent image of |CL| with
-the `mixer` bundle installed. If the bundle is not yet installed, you can
-add it with the :command:`swupd bundle-add` command as follows:
+To start working with the mixer tool, you need a recent image of |CL| with
+the `mixer` bundle installed. Before you install any new packages, update |CL|
+with the following command:
+
+.. code-block:: bash
+
+   sudo swupd update
+
+If the bundle is not yet installed, you can add it with the
+:command:`swupd bundle-add` command as follows:
 
 .. code-block:: bash
 
    sudo swupd bundle-add mixer
 
-Current mixing workflow
-***********************
-Mixer by default runs *all* build commands in a container to ensure the
+Mixer workflow
+**************
+Mixer by default runs all build commands in a container to ensure the
 correct version of the tooling is being used. This also allows custom mixes
 to automatically perform downstream format bumps when upstream releases
-a format bump. You can still run mixer natively by appending the *--native* flag to
-any of the commands. 
+a format bump. You can still run mixer natively by appending the *--native*
+flag to any of the commands.
 
 .. note::
-  You cannot run mixer if you are already in a container, unless you pass 
-  *--native* to the command. Nested containerization is not supported, nor 
+  You cannot run mixer if you are already in a container, unless you pass
+  *--native* to the command. Nested containerization is not supported, nor
   is building images using the container mode.
 
-There are two different workflows to create your own mix.
-First, if your mix only uses |CL| content, *skip Create custom RPMs* below. 
-Second, if your mix includes your own 
-:abbr:`RPMs (RPM Package Manager files)`, follow all steps below. 
+The workflow to create your own mix using mixer is outlined below. If your
+mix only uses |CL| content, *skip the Create custom RPMs* step.
 
-.. contents:: 
-   :local:
-   :depth: 1
-   :backlinks: top
-
-The following sections contain detailed information on every step of
-these workflows.
+#. :ref:`create-nginx-web-server`
+#. :ref:`create-workspace-mixer`
+#. :ref:`generate-starting-point-mix`
+#. :ref:`edit-builder`
+#. :ref:`create-custom-rpms`
+#. :ref:`manage-bundles`
+#. :ref:`build-chroots`
+#. :ref:`create-update`
+#. :ref:`create-image`
+#. :ref:`update-mix-version-info`
 
 .. _create-nginx-web-server:
 
-Create nginx web server to host mixer updates
-=============================================
+Set up nginx web server
```

Comment:
"Set up nginx web server" --> "Set up a web server"

nginx is just an example.  It can be any web server.